lib/Sema/CSApply.cpp

```diff
@@ -7037,6 +7037,7 @@ Expr *ExprRewriter::coerceToType(Expr *expr, Type toType,
   case TypeKind::Error:
   case TypeKind::InOut:
   case TypeKind::Module:
+  case TypeKind::MoveOnly:
```

Comment:
@xedin I imagine that we want to be able to do coercion from non-move only to move only. That is, I should be able to do something like this without having to do a move:

```
func foo() -> @_moveOnly Klass {
   Klass()
}
```

I am not going to implement that in this PR, but I am assuming that this is the place I would implement such a thing if we wanted to do so?